Pascal.py

```diff
@@ -0,0 +1,27 @@
+"""
+118. Pascal's Triangle
+Time Complexity - O(n^2)
+Space Complexity - O()
+for rows will be 0 to numRows-1
+    initialize a temp
+    for columns limit from 0 to rows
+        if column is 0 or row
+            append 1
+        else
+            output[row-1][column-1] + output[row-1][column]
+    append temp in output list
+    
+        """
+class Solution:
```

Comment:
Good Job! Diksha. Keep it up. Please add some comments in plain English to explain your approach.